README.md

```diff
@@ -2,13 +2,96 @@
 
 ![](Extras/img.jpg)
 
-## Python / Raspberry Pi interpretation of Bryan Boyer's Very Slow Movie Player  
+[![standard-readme compliant](https://img.shields.io/badge/readme%20style-standard-brightgreen.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
 
+Very Slow Movie Player using Python + Raspberry Pi
 
-Full writeup of this project here:   
-https://medium.com/@tomwhitwell/how-to-build-a-very-slow-movie-player-in-2020-c5745052e4e4
+## Background
 
-Bryan's original post here:  
-https://medium.com/s/story/very-slow-movie-player-499f76c48b62  
+In December 2018, Bryan Boyer posted [“Creating a Very Slow Movie Player”](https://medium.com/s/story/very-slow-movie-player-499f76c48b62), an essay about light and Brasília and architecture in which Boyer builds an e-paper display that shows films at 24 frames per hour, rather than 24 frames per second so it takes about a year to play the 142 minutes of 2001: A Space Odyssey.
 
+In August 2020, Tom Whitwell posted ["How to Build a Very Slow Movie Player for £120 in 2020"](https://debugger.medium.com/how-to-build-a-very-slow-movie-player-in-2020-c5745052e4e4), with instructions on how to build a VSMP with the new [7.5-inch, Raspberry Pi-compatible e-paper display from Waveshare](https://www.waveshare.com/product/displays/e-paper/epaper-1/7.5inch-e-paper-hat.htm).
 
+SlowMovie is the code that runs a VSMP on your Raspberry Pi.
+
+## Install
+
+**Note:** These installation instructions assume you have access to your Raspberry Pi and that you have the hardware set up properly. See the [Medium post](https://debugger.medium.com/how-to-build-a-very-slow-movie-player-in-2020-c5745052e4e4) for more complete instructions.
+
+SlowMovie requires [python3](https://www.python.org/). It uses [ffmpeg](https://ffmpeg.org/) via [ffmpeg-python](https://pypi.org/project/ffmpeg-python/) for video processing, and [PIL](https://python-pillow.org/) [FIXME is this the right link?] for image processing.
```

Comment:
Any ideas whether https://python-pillow.org/ is the right link for PIL that we're using?

Reply:
I believe that is the right library for Python 3. Way more info than I cared to know put apparently "PIL" was the python 2 version that has been abandoned. Pillow is the currently active Python3 version but you still use "import PIL" to maintain backward compatibility. Guess I learned something today.

Reply:
Hehe thanks for doing that research! Resolved in 98c1801